frontends/web/src/routes/settings/manage-accounts.tsx

```diff
@@ -132,20 +163,27 @@ class ManageAccounts extends Component<Props, State> {
         }
         return Object.entries(this.erc20TokenCodes)
             .map(([tokenCode, name]) => {
-                const active = activeTokens && activeTokens.includes(tokenCode);
+                const active = !!activeTokens && activeTokens.includes(tokenCode);
```

Comment:
are the !! really needed? i didn't see typescript complain about sth like this yet.

Reply:
IAccount says activeTokens can be undefined/missing is that correct?

https://github.com/digitalbitbox/bitbox-wallet-app/blob/staging-multiaccounts/frontends/web/src/api/account.ts#L39

![Screen Shot 2021-05-19 at 12 02 02 PM](https://user-images.githubusercontent.com/546900/118794640-157e3880-b89a-11eb-9ae6-04a06347f092.png)

The Toggle component wants a boolean, so Typescript complains, could either:
- keep as is
- update IAccounts interface (if backend always returns at least an empty array)
- move `!!` to Toggle, but I think `active ` should be a boolean and never undefined.